chart/kube-arangodb-crd/templates/webhook/deployment.yml

```diff
@@ -0,0 +1,104 @@
+{{ if .Values.webhook.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "kube-arangodb-crd.name" . }}
+  namespace: {{ .Release.Namespace }}
+{{- if .Values.webhook.annotations }}
+annotations:
+{{ toYaml .Values.webhook.annotations | indent 8 }}
+{{- end }}
+labels:
+  app.kubernetes.io/name: {{ template "kube-arangodb-crd.name" . }}
+  helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+  app.kubernetes.io/managed-by: {{ .Release.Service }}
+  app.kubernetes.io/instance: {{ .Release.Name }}
+  release: {{ .Release.Name }}
+spec:
+  replicas: {{ .Values.webhook.replicaCount }}
+  strategy:
+{{ toYaml .Values.webhook.updateStrategy | indent 8 }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ template "kube-arangodb-crd.name" . }}
+      helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+      app.kubernetes.io/managed-by: {{ .Release.Service }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ template "kube-arangodb-crd.name" . }}
+        helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+        app.kubernetes.io/managed-by: {{ .Release.Service }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        release: {{ .Release.Name }}
+{{- if .Values.webhook.annotations }}
+      annotations:
+{{ toYaml .Values.webhook.annotations | indent 16 }}
+{{- end }}
+    spec:
+      serviceAccountName: {{ template "kube-arangodb-crd.name" . }}
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
```

Comment:
Why do we need this?

Reply:
This will ensure that this container will run on amd64 only - it is recommended way how to do it (not by nodeSelector)